pkg/template/template.go

```diff
@@ -10,12 +10,20 @@ import (
 	"github.com/cyberark/conjur-oss-suite-release/pkg/changelog"
 )
 
-// UnifiedChangelogTemplateData stores all the data needed for generation of
-// templates
-type UnifiedChangelogTemplateData struct {
+// Component represents a suite component with all of its changelogs and relevant
+// pin data
+type Component struct {
```

Comment:
I was scratching my head about the earlier bits of code wondering what this is a component of. I think naming this struct `SuiteComponent` or `ReleaseComponent` or `XComponent` would add clarity.